articles/hooks/concepts/credentials-exchange-extensibility-point.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
Suggest merging this with the guide doc on using the credentials-exchange extensibility point as the introduction.